src/data/faq_de.json

```diff
@@ -131,6 +131,17 @@
                             "</ul>"
                         ]
                     },
+                    {
+                        "title": "Warum bekomme ich die Fehlermeldung 'Dieser QR-Code ist kein gültiges Impfzertifikat (VC_PREFIX_INVALID)', wenn ich versuche meinen Impf-QR-Code zu scannen?",
+                        "anchor": "vac_cert_index",
+                        "active": true,
+                        "textblock": [
+                            "Es gibt derzeit (Stand: 9 Juni 2021) noch keine Impfzertifikate mit QR-Codes, die mit der App eingescannt werden können.",
+                            "Das in der Vergangenheit und momentan ausgestellte Dokument ist trotz des QR-Codes kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung, die dem deutschen Infektionsschutzgesetz folgt, nicht aber den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat.",

```

Comment:
"Das in der Vergangenheit und momentan ausgestellte Dokument ist trotz des QR-Codes kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung, die dem deutschen Infektionsschutzgesetz folgt, nicht aber den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat."

This sentence is problematic, like the English one, because it is not clear whether the sentence is supposed to be referring to the document which the user is scanning in or whether it describes any documents from the past and present. I think it would be better to make it more general, so that would be:

"Ein in der Vergangenheit bzw. momentan noch ausgestellte Dokument ist trotz des QR-Codes kein gültiges digitales Impfzertifikat, sondern lediglich eine Impfbescheinigung, die dem deutschen Infektionsschutzgesetz folgt, nicht aber den aktuellen neuen abgestimmten europäischen Richtlinien für ein digitales Impfzertifikat."